_data/config.yml

```diff
@@ -1,2 +1,3 @@
 grpc_release_branch: release-0_14
+grpc_java_release_tag: v1.0.0-pre1
```

Comment:
Do you want to just flip this to v1.0.0 since this branch will only be live when 1.0 is?